ext/pdo_dblib/dblib_driver.c

```diff
@@ -143,60 +143,92 @@ static zend_long dblib_handle_doer(pdo_dbh_t *dbh, const char *sql, size_t sql_l
 	return DBCOUNT(H->link);
 }
 
-static int dblib_handle_quoter(pdo_dbh_t *dbh, const char *unquoted, size_t unquotedlen, char **quoted, size_t *quotedlen, enum pdo_param_type paramtype)
+static int dblib_handle_quoter(pdo_dbh_t *dbh, const char *unquoted, size_t unquoted_len, char **quoted, size_t *quoted_len, enum pdo_param_type param_type)
 {
+	const char *hex = "0123456789abcdef";
+	char *q;
+	*quoted_len = 0;
+	char is_ascii = 1;
+	char is_int = 1;
+
+	switch (PDO_PARAM_TYPE(param_type)) {
+		case PDO_PARAM_LOB:
```

Comment:
My understanding is that this type is meant for streams:
http://php.net/manual/en/pdo.lobs.php

I agree that PDO's types are a bit limiting, but it'll potentially confuse things to overload this type for this use case. Do you know how other PDO drivers handle this content? I haven't had experience with something like this. Another approach I've seen, with a different API, involved a data type specific to non-stream binary data.